src/test/java/org/broadinstitute/consent/http/service/DraftServiceTest.java

```diff
@@ -217,32 +85,4 @@ public StreamingDeserializer(String document, DraftStudyDataset meta) {
       this.meta = meta;
     }
   }
-
-  private void assertThinUser(User user) {
-    assertThat(user.getRoles(), is(nullValue()));
-    assertThat(user.getUserId(), is(notNullValue()));
-    assertThat(user.getInstitutionId(), is(nullValue()));
-    assertThat(user.getEraCommonsId(), is(nullValue()));
-  }
-
-  private Map<String, FormDataBodyPart> getRandomFiles(Integer count) {
-    Map<String, FormDataBodyPart> mapOfFiles = new HashMap<>();
-    IntStream.range(0, count)
-        .forEach(index -> {
-          String name = String.format("file%d", index);
-          mapOfFiles.put(name, getFormDataBodyPartMock(name));
-        });
-    return mapOfFiles;
-  }
-
-  private FormDataBodyPart getFormDataBodyPartMock(String name) {
-    FormDataBodyPart part = mock(FormDataBodyPart.class);
-    when(part.getName()).thenReturn(name);
-    when(part.getMediaType()).thenReturn(MediaType.MULTIPART_FORM_DATA_TYPE);
-    when(part.getValueAs(InputStream.class))
-        .thenReturn(new ByteArrayInputStream(EMPTY_JSON_DOCUMENT.getBytes()));
-    return part;
-  }
-
-
-}
+}
```

Comment:
It's true that most of `DraftService` are passthrough methods but note that this test doesn't cover most of the methods (only 60% line coverage). I usually like to add tests for trivial code in case more business logic is added in the future, to make it easier to extend the tests.

Reply:
Coverage added.

Reply:
![image](https://github.com/user-attachments/assets/6a3e6767-5adb-426f-9336-67fb3b4b242e)